ftShoe/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#/*
+# * Copyright (C) 2017 iCub Facility
+# * Authors: Luca Tagliapietra, Silvio Traversaro
+# * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+# */
+
+# Compile the plugin by default
+set(COMPILE_BY_DEFAULT ON)
+YARP_PREPARE_PLUGIN(ftshoe TYPE yarp::dev::ftshoeDriver
+                           INCLUDE ftshoeDriver.h
+			   CATEGORY device
+			   EXTRA_CONFIG WRAPPER=AnalogServer)
```

Comment:
Now the default compilation is enabled by the `DEFAULT ON` https://github.com/robotology/yarp/blob/master/cmake/YarpPlugin.cmake#L174 option.